templates/impala/search.html

```diff
@@ -16,7 +19,10 @@
     {{ search_form.platform }}
   {% endif %}
   <button class="search-button" type="submit" title="{{ _('Search') }}"
-         src="{{ static('img/zamboni/global/btn-search.png') }}"></button>
+         src="{{ static('img/zamboni/global/btn-search.png') }}"
+          style="text-indent:-999em;">search
```

Comment:
We'd prefer to not add inline CSS. There might be a util class already in our CSS that does this or similar.